popup.css

```diff
@@ -0,0 +1,202 @@
+/* *, *::after, *::before {
+    box-sizing: border-box;
+} */
+
+.modal {
+  color: #344563;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%) scale(0);
+  border: 1px solid black;
+  z-index: 10;
+  width: 315px;
+  max-height: 85vh;
+  min-width: 36%;
+  background-color: #fff;
+  transition: 200ms ease-in-out;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 16px;
+  justify-content: center;
+  overflow-y: scroll;
+}
+
+.modal.active {
+  transform: translate(-50%, -50%) scale(1);
+}
+
+.modal-title {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 7rem;
+  padding-bottom: 2rem;
+}
+
+.modal-title h2 {
+  font-size: 25px;
+  line-height: 44px;
+  font-weight: 700;
+}
+
+.modal-title .close-btn {
+  font-size: 1.6rem;
+  border: none;
+  outline: none;
+  background-color: transparent;
+  color: hsl(212, 16%, 48%);
+  cursor: pointer;
+  margin-left: 2rem;
+  opacity: 0.7;
+}
+
+.modal-text {
+  padding: 1rem 0;
+  font-size: 16px;
+}
+
+#overlay {
+  opacity: 0;
+  position: fixed;
+  inset: 0 0 0 0;
+  background-color: rgba(0, 0, 0, 0.5);
+  pointer-events: none;
+  transition: 200ms ease-in-out;
+}
+
+#overlay.active {
+  opacity: 1;
+  pointer-events: all;
+}
+
+.snap img {
+  width: 311px;
+  height: 220px;
+}
+
+.buttons {
+  display: flex;
+  flex-direction: row;
+  margin-right: 1.2rem;
+  border: 1px solid var(--White-color);
+}
+
+.buttons a {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  line-height: normal;
+  color: var(--White-color);
+  background-color: var(--Orange-color);
+  margin-right: 0.8rem;
+  padding: 10px 12px;
+  font-size: 15px;
+}
+
+.buttons a:hover {
+  color: var(--Orange-color);
+  background: var(--Text-color-main);
+  border: 1px solid var(--Text-color-main);
+}
+
+.buttons a:disabled {
+  background: hsl(15, 15%, 95%);
+  color: var(--Dark-color);
+}
+
+.buttons a:active {
+  color: hsl(15, 15%, 95%);
+}
+
+.buttons img {
+  width: 20%;
+}
+
+@media screen and (max-width: 374px) {
+  .modal .row-skills ul {
+    padding: 0.8rem 0;
+  }
+
+  .modal .row-skills li {
+    padding: 10px 7px;
+    margin-right: 0.3rem;
+  }
+
+  .modal {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 14px;
+    overflow-x: hidden;
+    width: 300px;
+    min-width: 50%;
+  }
```

Comment:
Your work on the desktop version of the popup is commendable. However, to align it with the Figma template, there are some minor issues that need attention:
- [ ] Please ensure that the title utilizes the same font family as specified in the template :+1:.
- [ ] To match the desired layout of the template, ensure that the technologies and title are properly aligned with the image :+1:.
- [ ] Remove any empty space present above the title to achieve consistency with the Figma design :+1: .

By addressing these minor issues, you will bring your popup design closer to the Figma template, resulting in a more cohesive and visually harmonious presentation. Kindly make the necessary adjustments accordingly.

**Your design**

![Screenshot from 2023-05-17 15-14-42](https://github.com/aradradev/abdulsportfolio/assets/105117832/002bedd4-b9c0-4d6d-bd89-bc56c0b10a17)

**Figma**

![Screenshot from 2023-05-17 15-15-01](https://github.com/aradradev/abdulsportfolio/assets/105117832/2c238506-2122-4cb0-a129-08aed09a0800)